tests/console/ipsec_tools_h2h.pm

```diff
@@ -0,0 +1,92 @@
+use base "consoletest";
+use lockapi;
+use testapi;
+use utils;
+use mmapi;
+use mm_network;
+
+sub run {
+    my $self = shift;
+    select_console 'root-console';
+    my $is_ipsec_primary = get_var('IPSEC_PRIMARY');
+
+    # Static IP address for Primary host and secondary host
+    my $ipsec_primary_ip   = "10.0.2.1/24";
+    my $ipsec_secondary_ip = "10.0.2.50/24";
+    my $my_static_ip       = $is_ipsec_primary ? $ipsec_primary_ip : $ipsec_secondary_ip;
+
+    # install ipsec-tools and config gateway and static ip on worker
+    zypper_call '-n in ipsec-tools';
+    configure_default_gateway;
+    configure_static_ip($my_static_ip);
+
+    if ($is_ipsec_primary) {
+        set_config($is_ipsec_primary, $ipsec_primary_ip, $ipsec_secondary_ip);
+        my $children = get_children();
+        my $child_id = (keys %$children)[0];
+        mutex_create('ipsec_primary');
+    }
+    else {
+        set_config($is_ipsec_primary, $ipsec_secondary_ip, $ipsec_primary_ip);
+        mutex_lock('ipsec_primary');
+        mutex_unlock('ipsec_primary');
+        my ($primary) = split('/', $ipsec_primary_ip);
+
+        # Test the secure channel, ping primary from secondary
+        assert_script_run "ping -c 6 $primary";
+    }
+}
+
+# Get pre-generated certs and config template for primary and secondary
+
+sub set_config {
+    my ($is_primary, $host_ip, $remote_ip) = @_;
+
+    # Split subnet mask
+    my ($host, $mask) = split('/', $host_ip);
+    my ($remote) = split('/', $remote_ip);
+    my $cert = $is_primary ? "server" : "client";
+
+    # Download pre-generated certs
+    my $setup_script = "curl -f -v " . autoinst_url . "/data/openssl/ca-cert.pem > /etc/racoon/cert/ca-cert.pem\n";
+    $setup_script .= "curl -f -v " . autoinst_url . "/data/openssl/$cert-cert.pem > /etc/racoon/cert/$cert-cert.pem\n";
+    $setup_script .= "curl -f -v " . autoinst_url . "/data/openssl/$cert-key.pem > /etc/racoon/cert/$cert-key.pem\n";
+
+    # Download config template
+    $setup_script .= "curl -f -v " . autoinst_url . "/data/racoon/racoon.conf > /etc/racoon/racoon.conf\n";
+    $setup_script .= "curl -f -v " . autoinst_url . "/data/racoon/setkey.conf > /etc/racoon/setkey.conf\n";
+    print $setup_script;
+    script_output($setup_script, 300);
+
+    # Fill in ip address to the config template
+    assert_script_run "sed -i 's/RemoteIP/$remote/' /etc/racoon/racoon.conf";
+    assert_script_run "sed -i 's/listenIP/$host/' /etc/racoon/racoon.conf";
+    assert_script_run "sed -i 's/RemoteIP/$remote\\\/$mask/' /etc/racoon/setkey.conf";
+    assert_script_run "sed -i 's/HostIP/$host\\\/$mask/' /etc/racoon/setkey.conf";
+    assert_script_run "sed -i 's/Remote/$remote/' /etc/racoon/setkey.conf";
+    assert_script_run "sed -i 's/Host/$host/' /etc/racoon/setkey.conf";
+
+    # Start racoon service
+    assert_script_run "systemctl start racoon.service";
+    assert_script_run "setkey -f /etc/racoon/setkey.conf";
+}
+
+sub test_flags {
+    return {important => 1};
```

Comment:
That flag does have no impact anymore. While you are at it, it would be appreciated if you also remove leftover references to that flag, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/search?p=1&q=important+%3D%3E&type=&utf8=%E2%9C%93